facade/nodeFacade.go

```diff
@@ -466,8 +466,8 @@ func (nf *nodeFacade) GetPeerInfo(pid string) ([]core.QueryP2PPeerInfo, error) {
 	return nf.node.GetPeerInfo(pid)
 }
 
-// GetConnectedPeersRatings returns the connected peers ratings
-func (nf *nodeFacade) GetConnectedPeersRatings() string {
+// GetConnectedPeersRatings returns the connected peers ratings on the main network
```

Comment:
If we add this comment we might consider renaming also the function to better state what it does:
GetConnectedPeersRatingsOnMainNetwork

Reply:
right, pushed